### PR TITLE
TID 2.0

### PR DESCRIPTION
**Pull request by jin0x**
This PR aims to implement the new design for istio.tetratelabs.io in accordance with the brand identity created by Made Studios, and to match styles with the current site tetrate.io

### CODE REVIEW

**Comment by dio** [comment]
The deploy preview is ready: https://deploy-preview-295--getistio-demo.netlify.app/.

**Review by PetrMc — APPROVED** [review]
LGTM

File: assets/js/form.js [hunk 1]
```diff
@@ -9,55 +9,63 @@ function validation() {
...
 
 function displayMessage() {
-  let emailForm = document.getElementById("emailFormContainer");
-  let formMessage = document.getElementById("formMessage");
+  let emailForm = document.getElementById('emailFormContainer');
+  let formMessage = document.getElementById('formMessage');
 
-  emailForm.className += " hidden";
-  formMessage.className += "active";
+  console.log('formMessage: ', formMessage);
```
Comment (dio) [inline assets/js/form.js]: Can we remove this `console.log`? (also in other places). Thanks. cc. @MB-Designs

Comment (jin0x) [reply assets/js/form.js]: Removed

**Comment by dio** [comment]
While we will develop a new page on the main site, I'm merging this for now.

**Pull request merged**